derp/emulator.py

```diff
@@ -107,9 +118,42 @@ def make_training_set(self, truth=None,
         
         self.Nts = self.X.shape[0]
         
+        # Do some pre-processing to remove NaNs, drop the IDs, and produce the 
+        # final X and y arrays to be stored in the class.
+        
+        self._preprocess_for_training()
+        
         return self.X, self.y
 
+    def _preprocess_for_training(self):
+        """
+        Internal helper function to clean up the training set.
+        """
+        # Find the object ID name:
+        X_ID_candidates, y_ID_candidates = [], []
```

Comment:
Is `object_id` not the finalized schema for a unique identifier? Is it bad if we hardcode this?

Reply:
I don't think we can hard code the object ID name - the EGC and DRP catalogs have different conventions. I guess there might be aliases? In any case, we may want to rename the object IDs so they have _different_ names (so we can keep track of which is from which catalog). But let's see if we can get by without doing this...